cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -557,6 +562,12 @@ private void configureStreamInput(Job job, StreamBatchReadable stream) throws IO
 
     job.setInputFormatClass(StreamInputFormat.class);
 
+    try {
```

Comment:
In that case too `configureStreamInput` is called - https://github.com/caskdata/cdap/blob/develop/cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java#L456